charts/incubator/terraria-tshock/questions.yaml

```diff
@@ -8,61 +8,168 @@ questions:
 # Include{replicas}
 # Include{replica1}
 # Include{controllerExpertExtraArgs}
-  - variable: env
-    group: "App Configuration"
-    label: "Image Environment"
+  - variable: terraria
+    group: App Configuration
+    label: Terraria Configuration
     schema:
       additional_attrs: true
       type: dict
       attrs:
-        - variable: GAME_PARAMS
-          label: "GAME_PARAMS"
-          description: "Enter your start up commands for the server."
+        - variable: pass
+          label: Server Pass
+          description: Sets the server password.
           schema:
             type: string
-            default: "-config serverconfig.txt"
+            private: true
+            required: true
+            default: ""
+        - variable: autocreate
+          label: Auto Create
+          description: Creates a world if none is found.
+          schema:
+            type: string
+            default: "1"
+            enum:
+              - value: "0"
+                description: None
+              - value: "1"
+                description: Small
+              - value: "2"
+                description: Medium
+              - value: "3"
+                description: Large
+        - variable: seed
+          label: World Seed
+          description: Specifies the world seed.
+          schema:
+            type: string
+            default: AwesomeSeed
+        - variable: worldname
+          label: Server World Name
+          description: Sets the name of the world
+          schema:
+            type: string
+            default: ""
+        - variable: motd
+          label: Server Message of The Day
+          description: Set the server motto of the day text.
+          schema:
+            type: string
+            default: ""
+        - variable: maxplayers
+          label: Max Players Count
+          description: Sets the max number of players.
+          schema:
+            type: int
+            default: 16
+            min: 1
+            max: 255
+        - variable: lang
+          label: Language
+          description: Sets the server's language.
+          schema:
+            type: string
+            default: en-US
+            enum:
```

Comment:
Those should be all required. Otherwise tpl will append commandline args with empty strings